### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by KaterynaEV**
- [DEMO LINK](https://KaterynaEV.github.io/layout_landing-page/)

### CODE REVIEW

**Review by xiimiik — CHANGES_REQUESTED** [review]
change to image with better resolution
image

add hovers here:
image
and for this links:
image
also disable scroll when menu is active (look at **checklist**)

**Comment by KaterynaEV** [comment]
added changes. landing is ok now?

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]
let's move back to previous comments from mentor:
1. you need to use svg icons instead of png
2. hover that you gave to the icons looks not really good. please just increase its size instead
3. hover's for links in menu is not like in design. please check

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great start! Consider fixing validation errors here
Screenshot 2023-10-15 at 16 33 13
Add a favicon
Screenshot 2023-10-15 at 16 31 29
Screenshot 2023-10-15 at 16 32 18
The phone icon shouldn't disappear on hover

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
1. Need to add the left and right indents for content
image

Keep going with the next sections

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,103 @@
+<!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bike</title>
```
Comment (denys-danyliuk) [inline src/index.html:7]:
```suggestion
    <title>MyBike</title>
```

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]
Good job! Next time request a review after you'll complete whole landing

**Review by etojeDenys — CHANGES_REQUESTED** [review]
looks broken on mobile
image
image

**Review by etojeDenys — CHANGES_REQUESTED** [review]
1. you should stretch the form on mobile
image


2. and make all form fields required

**Pull request closed without merging**